facia-json/src/main/scala/com/gu/facia/client/models/Config.scala

```diff
@@ -10,6 +10,33 @@ case class Backfill(
   query: String
 )
 
+sealed trait MetaData
+
+case object Tag1 extends MetaData
+
+case object Tag2 extends MetaData
+
+object MetaData {
+
+  implicit val tags = List("Tag1", "Tag2")
+
+  implicit object MetaDataFormat extends Format[MetaData] {
+    def reads(json: JsValue) = {
+      (json \ "type").transform[JsString](Reads.JsStringReads) match {
```

Comment:
It's better to stick to a `type` field in this case, as it is an object and not a string.

You could do this, but then how do you tell the difference between a string and an object, and how do you add more fields to the object (if you needed it).